components/dashboards/org.wso2.carbon.dashboards.api/src/main/java/org/wso2/carbon/dashboards/api/internal/DashboardRestApi.java

```diff
@@ -52,8 +56,8 @@
  * @since 4.0.0
  */
 @Component(name = "org.wso2.carbon.dashboards.api.DashboardApi",
-           service = Microservice.class,
-           immediate = true)
+        service = Microservice.class,
```

Comment:
Annotation parameters should be aligned. So shall we revert this formatting change, also below.

Reply:
Ack.